src/main/controllers/JudgmentDetailsController.ts

```diff
@@ -59,20 +59,37 @@ export default class JudgmentDetailsController {
         await updateDecisionState(selectedDecisionApplication.id, selectedDecision, req, logger);
       }
       const accessToken = req.session.user?.accessToken;
-      const selectedApplicationDocDownloadLink = await getApplicationDocDownloadLink(
-        selectedDecisionApplication,
-        logger,
-        accessToken,
-        res
-      );
-      const responseDocDownloadLink = await getResponseDocDownloadLink(
-        selectedDecisionApplication,
-        logger,
-        accessToken,
-        res
-      );
+      let selectedApplicationDocDownloadLink;
+
+      try {
+        selectedApplicationDocDownloadLink = await getApplicationDocDownloadLink(
+          selectedDecisionApplication,
+          accessToken
+        );
+      } catch (e) {
+        logger.error(e.message);
+        return res.redirect(ErrorPages.NOT_FOUND);
+      }
+
+      let responseDocDownloadLink;
+
+      try {
+        responseDocDownloadLink = await getResponseDocDownloadLink(selectedDecisionApplication, accessToken);
+      } catch (e) {
+        logger.error(e.message);
+        return res.redirect(ErrorPages.NOT_FOUND);
```

Comment:
Would be a good idea to query the language param here and redirect to Welsh page if necessary...and everywhere we do this redirect.

Reply:
I agree, but perhaps we should do this in another PR. I know a lot of 1.2 code has no regard for the languageParam yet
Perhaps this needs to be added to `et-quirks` and a ticket should be made to comb over everything 1.2 related for consistent languageParam use. There's another issue with 1.2 code not using "change" from translations files and pulling it in as the english literal

Reply:
I've updated the controllers I've touched with this languageParam. There's still merit in having a separate ticket to do a proper sweep of this. I believe we have a suggestion already to store the languageParam in the session instead of relying on the URL all of the time too